### PR TITLE
German translation updates for Git v2.21.0

### PR DESCRIPTION
**Pull request by ralfth**

### CODE REVIEW

File: po/de.po [hunk 1]
```diff
@@ -4661,6 +4649,16 @@ msgid ""
...
 msgstr ""
+"Das angegebene Ziel ist kein vollständiger Referenzname (startet mit \"refs/\").\n"
+"Wir versuchten zu erraten, was Sie meinten, mit:\n"
+"\n"
+"- Suche einer Referenz, die mit '%s' übereinstimmt, auf der Remote-Seite\n"
+"- Prüfung, ob die versendete <Quelle> ('%s') eine Referenz in \"refs/{heads,tags}\"\n"
+"  ist, in dessen Falle wir einen entsprechenden refs/{heads,tags} Präfix auf\n"
+"  der Remote-Seite hinzufügen würden.\n"
+"\n"
+"Keines hat funktioniert, so dass wir aufgegeben haben. Sie müssen die\n"
```
Comment (ruester) [inline po/de.po]: lieber "sodass" ?

File: po/de.po [hunk 2]
```diff
@@ -12111,185 +12095,181 @@ msgid "print only refs which don't contain the commit"
...
 msgid "%s: object corrupt or missing: %s"
-msgstr "Objekt nicht gefunden: %s"
+msgstr "%s: Objekt fehlerhaft oder nicht vorhanden: %s"
 
 #: builtin/fsck.c:585
-#, fuzzy, c-format
+#, c-format
 msgid "%s: object could not be parsed: %s"
-msgstr "konnte %s nicht parsen"
+msgstr "%s: Objekt konnte nicht geparsed werden: %s"
```
Comment (ruester) [inline po/de.po]: geparst ?

File: po/de.po [hunk 3]
```diff
@@ -17517,9 +17487,8 @@ msgid "--keep-unreachable and -A are incompatible"
 msgstr "--keep-unreachable und -A sind inkompatibel"
 
 #: builtin/repack.c:420
-#, fuzzy
 msgid "Nothing new to pack."
-msgstr "dieses Paket ignorieren"
+msgstr "Nichts neues zum Packen."
```
Comment (ruester) [inline po/de.po]: Neues

**Comment by ruester** [comment]
Acked-by: Matthias Rüster <matthias.ruester@gmail.com>

Thanks!

**Comment by ralfth** [comment]
Thanks for the review @ruester !

**Pull request merged**